wait/member.go

```diff
@@ -177,14 +175,13 @@ func (a *MemberAwaitility) WaitUntilNSTemplateSetDeleted(name string) error {
 }
 
 // WaitForNamespace waits until a namespace with the given owner (username), type, revision and tier labels exists
-func (a *MemberAwaitility) WaitForNamespace(username, typeName, revision, tier string) (*v1.Namespace, error) {
+func (a *MemberAwaitility) WaitForNamespace(username, ref string) (*v1.Namespace, error) {
 	namespaceList := &v1.NamespaceList{}
 	err := wait.Poll(a.RetryInterval, a.Timeout, func() (done bool, err error) {
 		namespaceList = &v1.NamespaceList{}
 		labels := map[string]string{
 			"toolchain.dev.openshift.com/owner":       username,
-			"toolchain.dev.openshift.com/type":        typeName,
```

Comment:
the `toolchain.dev.openshift.com/type` label should be still present. You can also add a check for the other labels:
* `toolchain.dev.openshift.com/tier`
* `toolchain.dev.openshift.com/provider`

Reply:
done in 4b9cc3d